spec.emu

```diff
@@ -331,6 +359,21 @@ contributors: Google, Ecma International
         The value of the [[RelevantExtensionKeys]] internal slot is &laquo;  &raquo;.
       </p>
 
+      <p>
+        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
+      </p>
+
+      <ul>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a types field for all locale values _locale_.  The value of this field must be a Record, which must have fields with the names of the six display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"dateTime"`, and `"dateTimeField"`.</li>
```

Comment:
According to tc39/ecma402#360, maybe this should be `a [[types]] field` instead of `a types field`.  @Ms2ger

Reply:
Also @leobalter @spectranaut

Reply:
I think so, yes; see also https://github.com/tc39/ecma402/pull/350.